src/packaging/files_core.txt

```diff
@@ -157,7 +157,7 @@ lib/qt/plugins/platforminputcontexts/libcomposeplatforminputcontextplugin.so [li
 lib/qt/plugins/platforminputcontexts/libibusplatforminputcontextplugin.so [linux]
 lib/qt/plugins/platforms/ [linux,mac]
 lib/qt/plugins/platforms/libqcocoa.dylib [mac]
-lib/qt/plugins/platforms/libqxcb.so [linux]
+lib/qt/plugins/platforms/lib*.so [linux]
```

Comment:
Is there an updated version of the snap that I can test on fedora?

Reply:
It's coming...